### ISSUE TITLE
Non Latin characters do not display properly

### ISSUE THREAD

**Issue by opensourcecheemsburgers** [opensourcecheemsburgers/RustyTube#15]
For example, inputting the following text into the search bar:
> 雷霆

Returns suggestions like: 
> `&#38647;&#38662;&#21152;&#36895;&#22120;`

This can be fixed by adding fonts which support these characters.

**Comment by opensourcecheemsburgers** [comment]
It seems to only be a problem for search bar suggestions. 

Pressing the search button with non-Latin characters entered will still return the correct results.

**Comment by opensourcecheemsburgers** [comment]
Upon closer inspection, Invidious actually responds with these html entities. 

For example: `https://invidious.protokolla.fi/api/v1/search/suggestions?q=小野正利` 

```
{
  "query": "&#23567;&#37326;&#27491;&#21033;",
  "suggestions": [
    "&#23567;&#37326;&#27491;&#21033;",
    "&#23567;&#37326;&#27491;&#21033; forever love",
    "&#23567;&#37326;&#27491;&#21033; &#24859;&#12434;&#12392;&#12426;&#12418;&#12393;&#12379;",
    "&#23567;&#37326;&#27491;&#21033; forever my love",
    "&#23567;&#37326;&#27491;&#21033; &#32418;",
    "&#23567;&#37326;&#27491;&#21033; &#12514;&#12491;&#12479;&#12522;&#12531;&#12464; &#29233;&#12434;&#12392;&#12426;&#12418;&#12393;&#12379;",
    "&#23567;&#37326;&#27491;&#21033; departure",
    "&#23567;&#37326;&#27491;&#21033; you're the only&#12539;&#12539;&#12539;",
    "&#23567;&#37326;&#27491;&#21033; &#12356;&#12388;&#12398;&#12414;&#12395;&#12363;&#21531;&#12434;",
    "&#23567;&#37326;&#27491;&#21033; destiny",
    "&#23567;&#37326;&#27491;&#21033; &#29454;&#20154;",
    "&#23567;&#37326;&#27491;&#21033; &#31048; &#27468;&#35422;",
    "&#23567;&#37326;&#27491;&#21033; cover"
  ]
}
```

Fixed in commit [`bdb1e98`](https://github.com/opensourcecheemsburgers/RustyTube/commit/bdb1e98fcdca5c2f71616a731f8c3458091e2ded).
